integration/shareddata/composites.go

```diff
@@ -40,6 +40,9 @@ var Composites = &Values[string]{
 		"array-three-reverse": bson.A{nil, "foo", int32(42)},
 		"array-empty":         bson.A{},
 		"array-null":          bson.A{nil},
+		"array-numbers-asc":   bson.A{int32(42), int32(43), int32(45)},
```

Comment:
@chilagrow as you mentioned in the call, those could be int32, int64, and float to check that we compare numbers correctly.

Reply:
fixed.